src/checkout-buttons/strategies/paypal/paypal-button-strategy.ts

```diff
@@ -34,12 +36,17 @@ export default class PaypalButtonStrategy extends CheckoutButtonStrategy {
             throw new InvalidArgumentError();
         }
 
-        return this._paypalScriptLoader.loadPaypal()
-            .then(paypal => {
+        return Promise.all([
+            this._paypalScriptLoader.loadPaypal(),
+            this._store.dispatch(this._checkoutActionCreator.loadDefaultCheckout()),
```

Comment:
It seems you only need the cart information inside the `payment` callback. Is it possible for you to make that call inside the callback instead? This way, we can render the button faster. And we only make the additional network work when the shopper clicks on the PayPal button.

Reply:
Yep, I was thinking the same thing after I left. I'll do that shortly.